concordium-contracts-common/src/schema_json.rs

```diff
@@ -956,13 +956,234 @@ impl Fields {
     }
 }
 
-/// Displays a pretty-printed JSON-template of the schema.
+/// Displays a pretty-printed JSON-template of the [`SchemaType`].
```

Comment:
```suggestion
/// Displays a pretty-printed JSON-template of the [`Type`].
```
`SchemaType` is the name of the derive macro :blush: